### PR TITLE
ref(ui): Use video for congrats robot animation. Remove lottie.

### PR DESCRIPTION
**Pull request by ckj**
Screen Shot 2019-09-30 at 11 57 01 AM

Todos:

- [ ] Get a higher resolution video that's not letterboxed

cc @maheskett

### CODE REVIEW

**Comment by maheskett** [comment]
This looks great :) 

I think `jest-canvas-mock` can be removed as well

**Comment by maheskett** [comment]
Updated the video to not have letterboxing. It's not higher resolution, but I think Hannah wanted to move forward with this one. cc @ckj 

Screen Shot 2020-01-21 at 1 37 24 PM

**Comment by ckj** [comment]
I can't approve since I created the PR, but LGTM.

File: package.json [hunk 1]
```diff
@@ -99,7 +99,6 @@
     "react-hot-loader": "4.12.16",
     "react-keydown": "^1.9.7",
     "react-lazyload": "^2.3.0",
-    "react-lottie": "^1.2.3",
```
Comment (EvanPurkhiser) [inline package.json]: Have to make sure we add this back into getsentry in that case, since it's needed for the animation in the upsell modal.

**Review by evanpurkhiser — APPROVED** [review]
Looks good!

File: package.json (hunk 1 above)
Comment (maheskett) [reply package.json]: Ah good thinking. It's already in getsentry separately https://github.com/getsentry/getsentry/blob/master/package.json#L37

**Pull request merged**